docs/edge.md

```diff
@@ -2,11 +2,13 @@
 
 This page explains how to use Edge Positioning module in your bot in order to enter into a trade only if the trade has a reasonable win rate and risk reward ratio, and consequently adjust your position size and stoploss.
 
-!!! Warning
+  !!! Warning
     Edge positioning is not compatible with dynamic (volume-based) whitelist.
 
-!!! Note
-    Edge does not consider anything else than buy/sell/stoploss signals. So trailing stoploss, ROI, and everything else are ignored in its calculation.
+  !!! Note
+ 1. Edge does not consider anything other than *its own* buy/sell/stoploss signals. It ignores the stoploss, trailing stoploss, and ROI settings in the strategy configuration file.
+
+ 2. Therefore, it is important to understand that Edge can improve the performance of some trading strategies but *decrease* the performance of others.
```

Comment:
I don't think this will render correctly. (`!!! Note` and the following, indented lines are rendered in special boxes in the documentation)

Once the indentation is broken (the empty line) - the box will end.

```suggestion
!!! Note
    Edge does not consider anything other than *its own* buy/sell/stoploss signals. It ignores the stoploss, trailing stoploss, and ROI settings in the strategy configuration file.
    Therefore, it is important to understand that Edge can improve the performance of some trading strategies but *decrease* the performance of others.
```

I'd not put it as numbered list, but simply as text - that should work out fine - considering it's rendered in a "note" box.
The text / wording itself is perfect though :+1: